cmd/cert-checker/main.go

```diff
@@ -109,12 +120,19 @@ func (c *certChecker) getCerts() error {
 	// so that we don't eat unnecessary amounts of memory and avoid the 16MB MySQL
 	// packet limit.
 	// TODO(#701): This query needs to make better use of indexes
+	q := getCertsQuery
+	args["limit"] = batchSize
+	if unexpiredOnly {
+		q += " AND expires >= :now"
+	}
+	q += " ORDER BY issued ASC LIMIT :limit OFFSET :offset"
```

Comment:
Similarly I think this can be merged into the main query if we make the change I suggest above.